### PR TITLE
(Deployment) py-atlas-building-tools: adds version 0.1.5 and removes unnecessary pin on voxcell version

### PR DESCRIPTION
**Pull request by lguyot**
Adds version 0.1.5 to `py-atlas-building-tools/package.yml` and removes unnecessary pin on the `voxcell` package version.

Deploying this new version will allow tasks pertaining to BBPP82-630 to be continued.

### CODE REVIEW

**Comment by lguyot** [comment]
@matz-e @pramodk Hi, sorry to bug you for this. Would it be possible to re-trigger a build? The git tag of the latest `atlas-building-tools` version was not available due to an unnoticed  failure of its gitlab build. Now that `atlas-building-tools-v0.1.5` does exit, every check should be successful.

**Comment by matz-e** [comment]
Retest this please

This is the way

File: var/spack/repos/builtin/packages/py-atlas-building-tools/package.py [hunk 1]
```diff
@@ -38,7 +39,7 @@ class PyAtlasBuildingTools(PythonPackage):
     depends_on('py-scikit-image@0.17.2:', type=('build', 'run'))
     depends_on('py-tqdm@4.44.1:', type=('build', 'run'))
     depends_on('py-trimesh@2.38.10:', type=('build', 'run'))
-    depends_on('py-voxcell@3.0.0', type=('run', 'build'))
+    depends_on('py-voxcell@3.0.0:', type=('run', 'build'))
```
Comment (eleftherioszisis) [inline var/spack/repos/builtin/packages/py-atlas-building-tools/package.py]: Out of curiosity, is the `build` type required in these dependencies? I was looking at the brainbuilder for example and it seems to specify all dependencies using the `run` type only:
https://github.com/BlueBrain/spack/blob/develop/var/spack/repos/builtin/packages/brainbuilder/package.py

**Comment by matz-e** [comment]
Retest this please

Hopefully, this will release that stuck Jenkins

File: var/spack/repos/builtin/packages/py-atlas-building-tools/package.py (hunk 1 above)
Comment (matz-e) [reply var/spack/repos/builtin/packages/py-atlas-building-tools/package.py]: See here: https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#python-dependencies, as noted by @GianlucaFicarelli in #1337 

Seems like that should actually be `build` and `run` to keep `pip` from downloading extra stuff.

**Review by matz-e — DISMISSED** [review]

File: var/spack/repos/builtin/packages/py-atlas-building-tools/package.py (hunk 1 above)
Comment (eleftherioszisis) [reply var/spack/repos/builtin/packages/py-atlas-building-tools/package.py]: That makes it clear. Thanks a lot!

**Comment by lguyot** [comment]
@matz-e Thanks for the review. Do you know why the last check is still pending? Should I trigger a new build?

**Comment by matz-e** [comment]
> @matz-e Thanks for the review. Do you know why the last check is still pending? Should I trigger a new build?

Kind of depends on core services to fix whatever is going on with Jenkins :(

**Review by eleftherioszisis — APPROVED** [review]

**Pull request merged**